installation_and_upgrade/ibex_install_utils/file_utils.py

```diff
@@ -192,11 +192,20 @@ def _get_dir_size(path="."):
         return total
     
     @staticmethod
-    def get_size(path='.'):
-        if os.path.isfile(path):
-            return os.path.getsize(path)
-        elif os.path.isdir(path):
-            return FileUtils._get_dir_size(path)
+    def get_size_and_number_of_files(path='.', ignore=None):
+        # use copytree to get size of directory
+        size_of_dir = 0
+        number_of_files = 0
+
+        def total_up_size(src, dst):
+            nonlocal size_of_dir
+            nonlocal number_of_files
+            number_of_files += 1
+            size_of_dir += os.path.getsize(src)
+        
+        shutil.copytree(path, 'C:/data', ignore=ignore, copy_function=total_up_size, dirs_exist_ok=True)
```

Comment:
I guess c:/data is not used, but can this be anything e.g. '' ?

Reply:
No it's not used for anything. I put C:/data as a just in case when I was testing it did do something. I will test now with just an empty string.